code/style.css

```diff
@@ -1,7 +1,250 @@
+}
+
+.hero-video {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  opacity: 0.75;
+}
+
+.nav-container {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 1em;
+}
+.branding {
+  height: 1.25rem;
+  object-fit: contain;
+}
+
+.menu-container {
+  display: flex;
+  z-index: 2;
+}
+.menu-link {
+  text-transform: uppercase;
+  color: var(--color-main);
+  font-weight: var(--fontweight-lato-regular);
+}
+.menu {
+  padding: 0.5em;
+  margin: 0.15rem;
+}
+
+.menu-link {
+  padding-bottom: 0.25em;
+  border-bottom: 1px solid transparent;
+}
+
+.menu-link:hover {
+  border-bottom: 1px solid var(--color-main);
+}
+
+.hero-branding {
+  z-index: 1;
+  width: 50vw;
+  object-fit: contain;
+  margin: 0 auto;
+}
+
+/* main */
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 1024px;
+  margin: 2rem auto;
+}
+
+.accordion,
+.form {
+  width: 25rem;
+  padding: 1em;
+  margin: 0.5rem;
+  min-height: 30rem;
+}
+
+/* form */
+.form {
+  background-color: var(--color-secondary);
+  display: flex;
+  flex-direction: column;
+}
+.space-divider {
+  height: 1rem;
+}
+.input-container.checkbox {
+  margin: 0.15rem 0;
+}
+.input-container {
+  display: inline-block;
+  margin: 0.25rem 0;
+  padding: 0.5em;
+  width: 27ch;
+}
+.checkbox-container.invalid,
+.input-container.invalid {
+  background-color: pink;
+}
+.label {
+  display: block;
+  margin-bottom: 0.25rem;
+}
+
+.input {
+  padding: 0.25em;
+  width: 100%;
+  margin-bottom: 0.25rem;
+}
+
+.radio-title {
+  padding-left: 0.5em;
+  margin-bottom: 0.25rem;
+}
+
+.radio-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0.25rem;
+  margin-left: 0;
+}
+
+.radio {
+  margin-right: 0.25rem;
+}
+.checkbox-container {
+  display: flex;
+  margin-bottom: 0.5rem;
+  padding: 0.5em;
+}
+.checkbox {
+  margin-right: 0.5rem;
+}
+
+.submit-container {
+  margin: 1.5rem auto;
+}
+.submit {
+  display: inline-block;
+  padding: 0.5em 2.5em;
+}
+
+/* accordion */
+.accordion {
+  background-color: var(--color-secondary);
+}
+.question {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  background-color: var(--color-indiePink);
+  padding: 0.25em;
+  margin-bottom: 0.25rem;
+}
+.icon-container {
+  width: 1.5rem;
+  height: 1.5rem;
+  padding: 0.25em;
+  border-radius: 50%;
+  border: 1px solid black;
+  display: flex;
+  margin-right: 0.5rem;
+}
+.icon {
+  width: 100%;
+  height: 100%;
+}
+.answer {
+  width: 100%;
+  display: none;
+  margin: 0;
+  margin-bottom: 1rem;
+  padding: 1em;
+  cursor: default;
+  background-color: var(--color-grey);
+}
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
+
+/* desktop first */
+@media screen and (max-width: 767px) {
+  .branding {
+    display: none;
```

Comment:
In desktop media query,
Min-width is set to 1024px;
In tablet min-width: 768px and max width: 1024px.
Can you check on this